delocate/tools/__init__.py

```diff
@@ -0,0 +1,44 @@
+import sys
+import os
+import zipfile
+
+from os.path import join as pjoin
+
+if sys.platform == 'win32':
```

Comment:
Not insisting, but it would be good to order these so it's obvious which names are specific to OSX / Windows, and which are not.